diff --git a/config.py b/config.py
index 838f205..ab06d01 100644
--- a/config.py
+++ b/config.py
@@ -1,18 +1,11 @@
-from petals.constants import PUBLIC_INITIAL_PEERS
-
 from data_structures import ModelInfo
 
-INITIAL_PEERS = PUBLIC_INITIAL_PEERS
+INITIAL_PEERS = ["/ip4/81.66.121.250/tcp/31337/p2p/QmXqqsBYNU7gmS4BaqPyADRYdNjmknhivRs36hUqCVqwye"]
 
 MODELS = [
     ModelInfo(
-        dht_prefix="StableBeluga2-hf",
-        repository="https://huggingface.co/petals-team/StableBeluga2",
-        num_blocks=80,
-    ),
-    ModelInfo(
-        dht_prefix="Llama-2-70b-hf",
-        repository="https://huggingface.co/meta-llama/Llama-2-70b-hf",
+        dht_prefix="Midnight-Miqu-70B-v1-5-hf",
+        repository="https://huggingface.co/sophosympatheia/Midnight-Miqu-70B-v1.5",
         num_blocks=80,
     ),
 ]
